### PR TITLE
[AUD-1627] Refactor EmptyCard to EmptyTile

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Refactors EmptyCard to EmptyTile and updates usages
- Fixes `you haven't` vs `userName hasn't` text for profile tabs

### CODE REVIEW

File: packages/mobile/src/components/core/EmptyTile.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    emptyTabContent: {
+      alignItems: 'center'
+    },
+    emptyCardTextRoot: {
+      flexDirection: 'row'
+    },
+    emptyCardTextEmoji: {
+      height: 20,
+      width: 20,
+      marginLeft: 4
```
Comment (Kyle-Shanks) [inline packages/mobile/src/components/core/EmptyTile.tsx:36]: spacing 😛

File: packages/mobile/src/screens/profile-screen/EmptyProfileTile.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+export const useEmptyProfileText = (
+  profile: Nullable<User>,
+  baseMessage: string
+) => {
+  const accountUser = useSelectorWeb(getAccountUser)
+
+  if (!profile) return ''
+  const { user_id, name } = profile
+  const isOwner = user_id === accountUser?.user_id
+  const youAction = `${messages.you} ${messages.haveNot}`
```
Comment (Kyle-Shanks) [inline packages/mobile/src/screens/profile-screen/EmptyProfileTile.tsx:23]: Wouldn't this cause you to have a message like 'You haven't haven't created any albums' if the emptyText passed in is 'haven't created any albums yet'?

**Review by Kyle-Shanks — CHANGES_REQUESTED** [review]
Looks good pending the "haven't haven't" issue

File: packages/mobile/src/screens/profile-screen/EmptyProfileTile.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/profile-screen/EmptyProfileTile.tsx:23]: yeah totally whoops, fixin

**Pull request merged**